css/style.css

```
@@ -0,0 +1,118 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+body {
+  margin: 0;
+  font-family: "Poppins";
+  display: grid;
+  grid-template: 0.15fr 0.85fr / 1fr 1fr 1fr;
+  background: #dc0a2d;
+}
+
+a{text-decoration: none;
+  color: inherit
+}
+
+header {
+  grid-area: 1/1/1/4;
+  padding: 4px;
+}
+
+.title {
+  display: flex;
+  align-content: center;
+  margin: 16px 20px 12px;
+}
+
+.title__image {
+  height: 100%;
+  align-self: center;
+}
+.title__text {
```

```suggestion
}

.title__text {
```

Pule uma linha aqui para ficar no padrão dos outros...